src/components/visualizations/chart/highcharts/plugins/getBubbleRadius.ts

```diff
@@ -0,0 +1,25 @@
+// (C) 2007-2018 GoodData Corporation
+import isNil = require("lodash/isNil");
+
+export function getBubbleRadius(Highcharts: any) {
```

Comment:
there is no highcharts interface available?

Reply:
`Highcharts` is defined as namespace then can not use it as `class` or `interface`. `any` can be used here.